.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+    ignorePatterns: ['!.*', '/coverage/', '/dist/', '/node_modules/', '/public/'],
+    extends: ['eslint:recommended', 'prettier'],
+    rules: {
+        '@typescript-eslint/explicit-module-boundary-types': 'off',
+        '@typescript-eslint/no-unsafe-argument': 'off',
+    },
+    env: {
+        node: true,
+    },
+    parser: '@typescript-eslint/parser',
+    parserOptions: {
+        ecmaVersion: 2022,
+        requireConfigFile: false,
+    },
+    reportUnusedDisableDirectives: true,
+    root: true,
+};
```

Comment:
We have company-wide ESLint configs you have to extend from. You probably want the typescript-node config from here: https://github.com/pickware/javascript-dev-tools/

The file should look like this:

```suggestion
module.exports = {
    ignorePatterns: ['!.*', '/coverage/', '/dist/', '/node_modules/', '/public/'],
    extends: [
        require.resolve('@pickware/javascript-dev-tools/eslint/configs/typescript-nestjs.js'),
    ],
    parserOptions: {
        project: require.resolve('./tsconfig.json'),
    },
    parser: '@typescript-eslint/parser',
    parserOptions: {
        ecmaVersion: 2022,
        requireConfigFile: false,
    },
    reportUnusedDisableDirectives: true,
    root: true,
};
```